.gitignore

```diff
@@ -81,3 +81,11 @@ lib/googletest/.github
 lib/googletest/googlemock/test
 lib/googletest/googletest/samples
 lib/googletest/googletest/test
+
+# Exclude unneeded files from the bundled hidapi library in the lib/
+# directory for future updates
+lib/hidapi/android
```

Comment:
wrong file?

Reply:
I have deleted the Android files and then added the directory to the excludes. What's wrong with this??

Reply:
Oh sorry, nothing. I originally thought we want to have individual .gitignore files in the sub-directory. But this way is working as well.